easybuild/easyconfigs/k/kalign2/kalign2-2.0.4-GCCcore-10.2.0.eb

```diff
@@ -0,0 +1,34 @@
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'ConfigureMake'
+
+name = 'kalign2'
+version = '2.0.4'
+
+homepage = ''
+description = "Kalign is a fast multiple sequence alignment program for biological sequences."
+
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['http://msa.sbc.su.se/downloads/kalign/']
+sources = ['current.tar.gz']
+checksums = ['8cf20ac4e1807dc642e7ffba8f42a117313beccaee4f87c5555d53a2eeac4cbb']  # current.tar.gz
+
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [
+    ('Autoconf', '2.69'),
+]
+
+configure_cmd_prefix = "autoreconf -vfi && "
```

Comment:
Is this actually necessary? Doesn't seem like it?

Reply:
Correct. I started with `kalign-3` first and used that working file as a template. This is where the came from. I removed it for the older version, it is needed for the GitHub version though. Thanks

Reply:
You missed removing autoconf dep

Reply:
Thanks. I removed it